### PR TITLE
add basic test of IDOM

### PR DESCRIPTION
**Pull request by rmorshea**

### CODE REVIEW

**Comment by rmorshea** [comment]
@Archmonger can you approave/merge this?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -32,4 +32,4 @@ jobs:
         run: |
           npm install -g npm@latest
           npm --version
-          nox -s test
+          nox -s test -- --headless
```
Comment (Archmonger) [inline .github/workflows/test.yml:35]: Was the double dash (`-- --headless`)intentional here?

**Comment by Archmonger** [comment]
> @Archmonger can you approave/merge this?

Awaiting one response on a potential typo. Let me know if it was intentional.

**Comment by rmorshea** [comment]
I think you forgot to publish the review.

**Review by Archmonger — DISMISSED** [review]

**Comment by Archmonger** [comment]
> I think you forgot to publish the review.

My bad, published.

File: .github/workflows/test.yml (hunk 1 above)
Comment (rmorshea) [reply .github/workflows/test.yml:35]: Yeah that was intentional - it delimits CLI args for `nox` vs args that end up a `Session.posargs` in `noxfile.py` (see [here](https://github.com/Archmonger/django-idom/pull/3/files#diff-f7a16a65f061822bcc73b8296f4dc837353d379d8d9cc5307982cb6941442835R54))

**Pull request merged**